src/main/java/global/govstack/threat_service/controller/impl/BroadcastControllerImpl.java

```diff
@@ -58,4 +58,13 @@ public ResponseEntity<BroadcastDto> publishBroadcast(UUID userId, BroadcastDto b
         }
         throw new UnauthorizedException("User doesn't have publish permission");
     }
+
+    @Override
+    public void deleteBroadcast(UUID broadcastId, UUID userId, int countryId) {
+        if (userService.canBroadcast(userId, countryId)) {
```

Comment:
Better to remove this validation for delete function completely, they're not compatible in any way.